### ISSUE TITLE
Composer does not install php sdk correctly

### ISSUE THREAD

**Issue by bunower** [directus/directus-sdk-php#30]
I cant install dirrectus-php-sdk correctly with following  requirements.
`
{
  "require": {
    "directus/sdk": "~1.0.0"
  },
  "minimum-stability": "dev",
  "repositories": [{
    "type": "git",
    "url": "https://github.com/wellingguzman/zend-db"
  }]
}
`
I get an error when i use the getitems function.

**Comment by WellingGuzman** [comment]
- You can't install the package correctly? are you getting any specific error from composer?
- What error are you getting when you use `getItems`? can you give us an example of your code?

**Comment by bunower** [comment]
The Problem is i am installing it correctly and the isntallation goes through.
So this is my `composer.json`

```
{

  "require": {
    "directus/sdk": "~1.0.0",
    "twig/twig": "^2.0"
  },
  "minimum-stability": "dev",
  "repositories": [{
    "type": "git",
    "url": "https://github.com/wellingguzman/zend-db"
  }]
}

```
This is my vendor directory after `composer install`  and `composer update`

```
-rw-r--r--  1 root root  183 Apr 19 13:38 autoload.php
drwxr-xr-x  2 root root 4096 Apr 19 13:37 composer
drwxr-xr-x 11 root root 4096 Apr 19 13:36 directus
drwxr-xr-x  5 root root 4096 Apr 19 13:36 guzzlehttp
drwxr-xr-x  3 root root 4096 Apr 19 13:36 league
drwxr-xr-x  3 root root 4096 Apr 19 13:36 psr
drwxr-xr-x  3 root root 4096 Apr 19 13:36 symfony
drwxr-xr-x  3 root root 4096 Apr 19 13:36 twig
drwxr-xr-x  4 root root 4096 Apr 19 13:36 zendframework
``` 
And this inside my directus folder : 

````
drwxr-xr-x  2 root root 4096 Apr 19 13:36 collection
drwxr-xr-x  2 root root 4096 Apr 19 13:36 config
drwxr-xr-x 10 root root 4096 Apr 19 13:36 database
drwxr-xr-x  3 root root 4096 Apr 19 13:36 filesystem
drwxr-xr-x  4 root root 4096 Apr 19 13:36 hash
drwxr-xr-x  2 root root 4096 Apr 19 13:36 hook
drwxr-xr-x  3 root root 4096 Apr 19 13:36 permissions
drwxr-xr-x  4 root root 4096 Apr 19 13:36 sdk
drwxr-xr-x  4 root root 4096 Apr 19 13:36 utils
```
Now after i include autoload try to use:

```
 $database = \Directus\SDK\ClientFactory::create($sdk_config);
 $database->getItem('page',1);

error:
Fatal error: Uncaught Error: Call to undefined function Directus\Database\TableGateway\get_columns_flat_at() in /var/www/html/XXXXXXXXXXXXX.com/directus-sdk-php/vendor/directus/database/TableGateway/RelationalTableGateway.php:880
```

it only work when i also include the autoload of  directus.

**Comment by bunower** [comment]
Now i get the Error:

```
Uncaught Error: Call to undefined method Directus\Util\ArrayUtils::deepLevel() 
```

**Comment by WellingGuzman** [comment]
This has been fixed in the latest version `1.1.1`.
